test/helpers/compile.js

```diff
@@ -10,6 +10,7 @@ function compile(fixture, moduleRules, resolveAlias = {}) {
 
     webpack({
       entry,
+      mode: 'development',
```

Comment:
Please drop this as the test setup needs additional changes to work with `webpack >= v4.0.0` (`CommonChunksPlugin` removed etc etc) :) This will be taken care of by #233